lib/resty/rediscluster.lua

```diff
@@ -65,6 +67,81 @@ local cluster_invalid_cmds = {
 }
 
 
+local CACHED_SHA256
+
+
+local function hash_password(password)
+  if not password or password == "" then
+    return ""
+  end
+
+  local sha256, err
+  if CACHED_SHA256 then
+    sha256 = CACHED_SHA256
+
+  else
+    sha256, err = digest.new("sha256")
+    if not sha256 then
+      return nil, err
+    end
+    CACHED_SHA256 = sha256
```

Comment:
As this is only invoked at the very beginning of `new()`, should we cache the object?

Reply:
Every time Kong connects to redis it will invoke this `new()`, so I think it's not bad to cache the object.